### DISCUSSION TITLE
Thoughts on adding Golang

### DISCUSSION THREAD

**Discussion by XurifyTeine** [nhost/nhost#1990] [Ideas]
Would like to know your thoughts on adding Golang as an option to launch projects written in

**Comment by dbarrosop** [comment]
Would you mind clarifying what you mean with that? Are you talking about serverless functions or fully fledged services?

**Reply by gitekDev** [reply]
i think he means serverless functions to start with a runtime (dotnet, golang, python .. ) like appwrite.
I dont use appwrite because (mariadb), but they have somethink like that to start functions with runtime select, select scheudle and go with it. 

Somethink like that would be a cool future at nhost maybe?

**Reply by XurifyTeine** [reply]
Yes, exactly @xoose ! The ability to deploy serverless functions in golang

(Sorry for the late response, didn't see the notification until now)
